### PR TITLE
@xtina-starr => add ENV force server events bucket key #migration

### PR DESCRIPTION
**Pull request by erikdstock**
This PR adds a new environment variable, `SEGMENT_WRITE_KEY_SERVER` for a new force server events bucket. The key is used by @artsy/passport v1.0.8 for tracking account creations. It also makes a minor change requested by @damassi and one file seems to have been formatted by prettier because I touched it.

When we are ready to merge I'll set the staging bucket environment variable, verify events are coming through as expected and then create the production bucket to complete this process.

### CODE REVIEW

File: .env.oss [hunk 1]
```diff
@@ -39,6 +39,7 @@ S3_BUCKET=
 SAILTHRU_KEY=
 SAILTHRU_SECRET=
 SEGMENT_WRITE_KEY=
+SEGMENT_WRITE_KEY_SERVER=
```
Comment (mzikherman) [inline .env.oss]: I'm not sure why we have all these non-OSS placeholders in `.env.oss`, seems odd.

**Comment by mzikherman** [comment]
Mentioned in the passport PR, but I think this should just be: https://github.com/artsy/force/blob/89957b4d2832301bc0b37bf07f4ad01b84f5bfcc/src/lib/setup.js#L146-L151

```
artsyPassport(
      _.extend({}, config, {
        CurrentUser: CurrentUser,
        ARTSY_URL: API_URL,
        SEGMENT_WRITE_KEY: SEGMENT_WRITE_KEY_SERVER
        userKeys: [
          'collector_level',
        ....
```

**Comment by erikdstock** [comment]
excellent, thank you! That thought only occurred to me as I was getting these PRs together.

File: .env.oss (hunk 1 above)
Comment (erikdstock) [reply .env.oss]: I believe this was just a result of the desktop + mobile consolidations- getting the two environments together.

File: src/lib/setup.js [hunk 2]
```diff
@@ -220,7 +222,7 @@ export default function(app) {
     })
 
     mountAndReload(path.resolve('src/desktop'), {
-      watchModules: ['@artsy/reaction', '@artsy/stitch'],
+      watchModules: ['@artsy/reaction'],
```
Comment (orta) [inline src/lib/setup.js]: /cc @damassi ^

Comment (damassi) [reply src/lib/setup.js]: Thnx @erikdstock - this `watchModules` field should be used sparingly lest someone accidentally wire up a local version of a module that is in an old state. I realized I had left stitch in there from before when we were SHing earlier.

File: .env.oss (hunk 1 above)
Comment (alloy) [reply .env.oss]: Listing all env variables being used by a program is just _a_ way to document those. After cloning the repo and copying this file to `.env` it should be clear to the dev what configuration needs to be done.

**Comment by xtina-starr** [comment]
This looks pretty straight forward to me. I'm going to go ahead and merge.

File: .env.oss (hunk 1 above)
Comment (mzikherman) [reply .env.oss]: I get that, I guess I'm just musing if there's a better way. For instance @izakp and I briefly discussed something similar with regards to Metaphysics secrets, plus https://github.com/artsy/post_mortems/issues/28 (caused by leaky `.env` file), so I figured I'd raise a PR comment/question.

Agreed that its a pattern we follow, who knows? Maybe that'll change someday...

Comment (alloy) [reply .env.oss]: Oh I definitely think the consolidated config file is a better approach.

**Pull request merged**